components/DaoMenu.tsx

```diff
@@ -46,15 +46,15 @@ const DaoMenu = <T,>(props: Props<T>) => {
       >
         <Menu.Items className="absolute right-0 z-10 mt-2 w-44 origin-top-right rounded-md bg-white shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none">
           <div className="py-1">
-            {props.options.map((option, key) => (
-              <Menu.Item key={key}>
+            {props.options.map((option, index) => (
+              <Menu.Item key={index}>
                 {({ active }) => {
                   return (
                     <Link
-                      href={`/${option.value}/bridge`}
+                      href={`/${option.daoId}/bridge`}
```

Comment:
This may come up in the resolving  the merge conflict. We shoudl redirect to the same tab the user is on.

Reply:
done! thanks.